src/common/dependencies.ts

```diff
@@ -43,24 +44,52 @@ const getOnnxRuntime = (canCuda: boolean): PyPiPackage => {
     };
 };
 
-export const getOptionalDependencies = (isNvidiaAvailable: boolean): Dependency[] => {
+const getPytorch = (canCuda: boolean, pythonVersion: Version): PyPiPackage[] => {
+    if (semver.gte(pythonVersion, '3.8.0') && semver.lt(pythonVersion, '3.10.0')) {
```

Comment:
Our minimum supported python version is 3.8.0, so won't `semver.gte(pythonVersion, '3.8.0')` always be true?

Reply:
Yes, that's why i made it there an error otherwise.

Reply:
Fair.